src/pages/ApplicationConfig/components/TriggerDialog/index.tsx

```diff
@@ -330,23 +330,23 @@ class TriggerDialog extends React.Component<Props, State> {
             </Col>
 
             <Col span={12} style={{ padding: '0 8px' }}>
-              <If condition={this.field.getValue('payloadType') !== 'custom'}>
-                <FormItem label={<Translation>Component</Translation>} required>
-                  <Select
-                    name="componentName"
-                    locale={locale().Select}
-                    dataSource={componentsOption}
-                    {...init('componentName', {
-                      rules: [
-                        {
-                          required: true,
-                          message: 'Please select a component',
-                        },
-                      ],
-                    })}
-                  />
-                </FormItem>
-              </If>
+              <FormItem label={<Translation>Component</Translation>} required>
+                <Select
+                  name="componentName"
+                  locale={locale().Select}
+                  dataSource={componentsOption}
+                  {...init('componentName', {
+                    initValue: components.length > 0 && components[0].name,
+                    rules: [
+                      {
+                        required: true,
+                        message: 'Please select a component',
```

Comment:
could you to do international translation?

Reply:
This component is unuseful, I will delete it.